easybuild/easyconfigs/y/Yambo/Yambo-3.4.2-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'Yambo'
+version = '3.4.2'
+
+homepage = 'http://www.yambo-code.org'
+description = """Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.
+ Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+dependencies = [('netCDF-Fortran', '4.4.2')]
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['http://qe-forge.org/gf/download/frsrelease/208/932/yambo-3.4.2.tgz']
+checksums = ['f0820a0f42dfc81ce3811af647bed777']
+
+configopts = 'CPPFLAGS="" FCFLAGS="-nofor_main" --with-blas-libs="$LIBBLAS" '
+configopts += '--with-lapack-libs="$LIBLAPACK" --with-blacs-libs="$LIBBLACS" '
+configopts += '--with-scalapack-libs="$LIBSCALAPACK" --with-fft-libs="-mkl" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff -L$EBROOTNETCDF/lib64 -lnetcdf"'
```

Comment:
I'm not sure if the `/lib` and `/lib64` will always be correct here...

but, since the modules will already append these paths to `$LIBRARY_PATH`, I think you can just drop the `-L` flags?